index.html

```diff
@@ -1,42 +1,91 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
-    <!-- Bootstrap Styles-->
-    <link
-      rel="stylesheet"
-      href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
-      integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh"
-      crossorigin="anonymous"
-    />
-
-    <!-- INCLUDE STYLES FILE HERE -->
-  </head>
-
-  <body>
-    <div class="container">
-      <!-- WORK FROM INSIDE THIS CONTAINER-->
+
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title> Parneet Kang</title>
+  <!-- Bootstrap Styles-->
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+    integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous" />
+
+  <!-- INCLUDE STYLES FILE HERE -->
+  <link rel="stylesheet" href="resources/styles/styles.css" />
+</head>
+
+<body>
+  <div class="container">
+    <!-- WORK FROM INSIDE THIS CONTAINER-->
+    <h1> Parneet Kang</h1>
+
+    <img src="resources/images/Website.png" />
+    <div>
+      <div class="section" id="personalBackground">
+
+        <h2> Personal Background</h2>
+
+        <p>My name is Parneet Kang. I am majoring in Information Technology in Software Development
+          Track.My hobbies are traveling, trying new food and spending time with my family and friends. I am really
+          for this course and it seems like I will learn many new things from this course.
+        </p>
+      </div>
+
+      <div class="section" id="experience">
+        <h2>Work and Development Experience</h2>
+        <p>I have learnt HTML and CSS from Web Development class. I have also made an app
+          with the help of my group in the last semester. From there, I am little familiar with
+          GitHub. All the experience I have with web development is from in class work done.
+        </p>
+      </div>
+
+      <div class="section" id="programmingLanguages">
+        <h2>Programming Languages</h2>
+        <ol>
+          <li>Java</li>
+          <li>C#</li>
+          <li>HTML</li>
+          <li>Kotlin</li>
+          <li>SQL Database</li>
+          <li>Linux</li>
+        </ol>
+      </div>
+
+      <div class="section" id="achievements">
+        <h2>Achievements</h2>
+        <ul>
+          <li>Dean List</li>
+          <li>Passed all the classes in Spring 2020</li>
```

Comment:
and what an achievement! This wasn't an easy transition for anyone! Congratulation!